scalding-parquet-scrooge/src/test/scala/com/twitter/scalding/parquet/scrooge/ScroogeReadSupportTests.scala

```diff
@@ -90,6 +90,384 @@ class ScroogeReadSupportTests extends WordSpec with Matchers with HadoopSharedPl
     }
   }
 
+  "ScroogeReadSupport resolving list format" should {
```

Comment:
Note that in projected schema, the field is optional, and that is maintained in the result.